__tests__/index.test.js

```diff
@@ -181,3 +183,78 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("POST:201 inserts a new comment to articles and sends the new comment back", () => {
+    const newComment = {
+      author: "butter_bridge",
+      body: "I am 100% sure that we're not completely sure.",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.comment.comment_id).toBe(19);
+        expect(response.body.comment.body).toEqual("I am 100% sure that we're not completely sure.");
+        expect(response.body.comment.author).toEqual("butter_bridge")
+        expect(response.body.comment.article_id).toBe(9);
+        expect(response.body.comment.votes).toBe(0);
+       });
+    });
+  test("POST:201 inserts a new comment to articles and sends the new comment back with a property that doesn't exist", () => {
```

Comment:
Please edit these comments to make the descriptions more clear.  The tests are fine, just descriptions could be better!